SAMtags.tex

```diff
@@ -80,6 +81,7 @@ \section{Standard tags}
   {\tt MC} & Z & CIGAR string for mate/next segment\\
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
+  {\tt MI} & Z & Molecular-barcode (corrected) sequence or identifier. May contain non sequence bases to encode more information (e.g., strandedness of original molecule) \\
```

Comment:
@nh13 and I introduced this tag in [fgbio](https://github.com/fulcrumgenomics/fgbio) several months ago.  The intended purpose is _not_ to contain sequence bases at all, but instead to contain a compact identifier that uniquely identifies the source molecule within the entire BAM without reference to other fields.  In our usage we just use serial numbers as identifiers.

I'd suggest changing the language to something like: `Molecular Identifier.  A unique ID within the SAM file for the source molecule from which this read is derived.  All reads with the sam MI tag represent the group of reads derived from the same source molecule.`